tap_hubspot_beta/streams.py

```diff
@@ -1095,7 +1095,7 @@ class ListMembershipV3Stream(hubspotV3Stream):
 
     schema = th.PropertiesList(
         th.Property("results", th.CustomType({"type": ["array", "string"]})),
-        th.Property("list_id", th.IntegerType),
+        th.Property("list_id", th.StringType),
```

Comment:
This is coming from the parent stream, and the parent stream has the type set to

```py
th.Property("listId", th.NumberType()),
```

so will this change work?